### ISSUE TITLE
support for Tensorflow version 2

### ISSUE THREAD

**Issue by dremsol** [dusty-nv/jetson-containers#11]
Does Jetson support TF2 such that it can be supported in a container?

The following [Release Notes](https://docs.nvidia.com/deeplearning/frameworks/install-tf-jetson-platform-release-notes/tf-jetson-rel.html#tf-jetson-rel) suggest this should be supported. However this mentions the complete TF container which is almost double the size.

**Comment by dusty-nv** [comment]
Yes, the wheels are available and you could install it into containers. I just used TF 1.15 in these initial versions.
________________________________

**Comment by dremsol** [comment]
Thanks, would you like me to create a PR here? In that case are those wheels also available at following endpoint?

`https://nvidia.box.com/shared/static/`
